core/platform/models_test.py

```diff
@@ -204,11 +226,13 @@ def test_import_models_user(self):
             self.registry_instance.import_models([models.NAMES.user]))
 
     def test_import_models_invalid(self):
+        # type: () -> None
         """Tests import_models function with an invalid option."""
-        with self.assertRaisesRegexp(Exception, 'Invalid model name: '):
+        with self.assertRaisesRegexp(Exception, 'Invalid model name: '): # type: ignore[no-untyped-call]
```

Comment:
Will this ignore be removed after test_utils has been annotated?

Reply:
Yes. This is due to assertRaisesrRgexp being untyped.